developer-guides/rest-api/README.md

```diff
@@ -166,6 +166,7 @@ When calling a production Rocket.Chat server, ensure it is running via HTTPS and
 | `/api/v1/chat.unStarMessage`          | Removes the star on the chat message for the authenticated user. | [Link](chat/unstarmessage/)             |
 | `/api/v1/chat.update`                 | Updates the text of the chat message.                            | [Link](chat/update/)                    |
 | `/api/v1/chat.getMessageReadReceipts` | Retrieves message read receipts.                                 | [Link](chat/getmessagereadreceipts/)    |
+| `/api/v1/chat.ignoreUser`             | Ignores an user from a chat.                                     | [Link](chat/ignoreuser/)                |
```

Comment:
@thayannevls can you please put this in alphabetic order? Also if you can put the `getMessageReadReceipts` in order, because it is unordered too.

Reply:
@thayannevls I think that here, you forgot to put in alphabetic order :no_mouth:

Reply:
Oh sorry 😅 now is fixed